lib/audit/test/audit.js

```diff
@@ -1,3 +1,4 @@
+const moment = require('moment')
```

Comment:
At Tyler's recommendation we're switching to [date-fns](https://date-fns.org/) instead of moment since it is significantly lighter weight.